### PR TITLE
Change writer to author where necessary

### PR DESCRIPTION
**Pull request by amrandahvur**
RELEASE_NOTES.rst: Change writer->author

coalib/bears/Bear.py: Change writer->author

Change of word writer to author wherever necessary

Closes https://github.com/coala/coala/issues/2707

### CODE REVIEW

**Comment by Nosferatul** [comment]
Awesome!

File: RELEASE_NOTES.rst [hunk 1]
```diff
@@ -17,32 +17,32 @@ coala 0.8.0 - grizzly
...
-       mMMMMMy``` dMMMMMMMMMMMMMh.`sMMMh`   
-       yMMMMMMy  `MMMMMMMMMMMMMMMMy:..`     
-       `yMMMMMd  yMMMMMMMMMMMMMMMMMMMMNh+`  
-         .ohhs-+mMMMMMMMMMMMMMMMMMMMMMMMMd  
-            .yMMMMMMMMMMMMMMMMMMMMMMMMMMMh  
-            mMMMMMMMMMMMMMMMMMMMMMMMMMMMh`  
-            yMMMMMMMMMMMMMMMNhssssyyyso-    
-             /dMMMMMMMMMNy+.                
-               ./syhys/-                    
+                   -
```
Comment (Adrianzatreanu) [inline RELEASE_NOTES.rst]: im not sure why this happened, but i dont think this change belongs to this commit, so please try to leave it as it was :D

File: RELEASE_NOTES.rst [hunk 2]
```diff
@@ -17,32 +17,32 @@ coala 0.8.0 - grizzly
 
 ::
```
Comment (Adrianzatreanu) [inline RELEASE_NOTES.rst]: also in commit shortlog please enclose "writer" and "author" in double backticks


and add a period at the end of the commit body.

File: RELEASE_NOTES.rst (hunk 1 above)
Comment (amrandahvur) [inline RELEASE_NOTES.rst]: So I tried resolving this, but for some reason, my editor automatically deletes and regenerates it. Should it be a problem considering there is no final effect on it ?

File: RELEASE_NOTES.rst (hunk 2 above)
Comment (amrandahvur) [inline RELEASE_NOTES.rst]: Sure, Working on it :D

File: RELEASE_NOTES.rst (hunk 1 above)
Comment (Adrianzatreanu) [inline RELEASE_NOTES.rst]: try another editor :p ? work on settings from editor?

**Comment by amrandahvur** [comment]
Adding another PR (to resolve first issue of regeneration of logo)

File: RELEASE_NOTES.rst (hunk 1 above)
Comment (amrandahvur) [inline RELEASE_NOTES.rst]: Fixed and opened a new pull request for this #2851

**Pull request closed without merging**